samples/star-wars-api/HttpHandlers.fs

```diff
@@ -60,9 +60,22 @@ module HttpHandlers =
 
             let removeWhitespacesAndLineBreaks (str : string) = str.Trim().Replace ("\r\n", " ")
 
+
+            let hasData (context: HttpContext) =
+                let request = context.Request
+                if request.Body.CanSeek then
+                    request.Body.Length > 0L
+                else
+                    ctx.Request.EnableBuffering()
+                    let buffer = Array.zeroCreate 1
+                    let value = request.Body.Read(buffer, 0, 1)
+                    ctx.Request.Body.Seek(0, SeekOrigin.Begin) |> ignore
+                    value > 0
+
             // TODO: Figure out how to check if body is empty
             // TODO: Return introspection on GET
-            if ctx.Request.Body.Length = 0
+            //http context replace network stream with buffered stream
+            if (hasData ctx = false || ctx.Request.Method = "GET")
```

Comment:
```suggestion
            if (ctx.Request.Method = "GET" || hasData ctx = false)
```
I think the first condition is "GET"
There should be a constant for "GET", let's use it